src/query/storages/fuse/src/table_functions/fuse_snapshots/fuse_snapshot.rs

```diff
@@ -62,12 +62,12 @@ impl<'a> FuseSnapshot<'a> {
         let mut snapshot_locations: Vec<Vec<u8>> = Vec::with_capacity(len);
         let mut prev_snapshot_ids: Vec<Option<Vec<u8>>> = Vec::with_capacity(len);
         let mut format_versions: Vec<u64> = Vec::with_capacity(len);
-        let mut segment_count: Vec<u64> = Vec::with_capacity(len);
-        let mut block_count: Vec<u64> = Vec::with_capacity(len);
-        let mut row_count: Vec<u64> = Vec::with_capacity(len);
-        let mut compressed: Vec<u64> = Vec::with_capacity(len);
-        let mut uncompressed: Vec<u64> = Vec::with_capacity(len);
-        let mut index_size: Vec<u64> = Vec::with_capacity(len);
+        let mut segment_count: Vec<u32> = Vec::with_capacity(len);
```

Comment:
@BohuTANG  This is incorrect, the schema will be mismatch. 

Fixed in https://github.com/datafuselabs/databend/pull/8154

with commit https://github.com/datafuselabs/databend/pull/8154/commits/6d135bd7cff3a180d60d3fce892b840eca8d6997